Taarafo.Core.Tests.Unit/Services/Foundations/Comments/CommentServiceTests.Exceptions.Add.cs

```diff
@@ -194,5 +194,52 @@ public async Task ShouldThrowServiceExceptionOnAddIfDatabaseUpdateErrorOccursAnd
             this.storageBrokerMock.VerifyNoOtherCalls();
             this.loggingBrokerMock.VerifyNoOtherCalls();
         }
+
+        [Fact]
+        public async void ShouldThrowValidationExceptionOnAddIfReferenceErrorOccursAndLogItAsync()
+        {
+            // given
+            DateTimeOffset randomDateTime =
+                GetRandomDateTimeOffset();
+
+            Comment randomComment = CreateRandomComment(randomDateTime);
+            Comment someComment = randomComment;
```

Comment:
For creating `someEntity` you don't need to create `randomEntity`

Reply:
@hassanhabib I removed the randomComment